### PR TITLE
*: Tiny cleanup

### PR DESCRIPTION
**Pull request by shenli**

### CODE REVIEW

File: perfschema/statement_test.go [hunk 1]
```diff
@@ -29,7 +29,7 @@ func (p *testStatementSuit) TestUninitPS(c *C) {
 	// Run init()
 	ps := &perfSchema{}
 	// ps is uninitialized, all mTables are missing.
-	// This cound happend at the bootstrap stage.
+	// This cound happened at the bootstrap stage.
```
Comment (disksing) [inline perfschema/statement_test.go]: cound -> could?

Comment (shenli) [inline perfschema/statement_test.go]: Good catch!

File: util/hack/hack_test.go [hunk 2]
```diff
@@ -36,6 +36,8 @@ func TestString(t *testing.T) {
 	if a != "aello world" {
 		t.Fatal(a)
```
Comment (zimulala) [inline util/hack/hack_test.go]: `t.Fatal(a)` -> `t.Fatalf("a:%v, b:%v", a, b)` to solve the ineffassign warning ?

**Comment by qiuyesuifeng** [comment]
LGTM

**Comment by shenli** [comment]
@zimulala PTAL

**Comment by shenli** [comment]
PTAL @qiuyesuifeng

**Comment by zimulala** [comment]
LGTM

**Pull request merged**